triton-server/config.ts

```diff
@@ -15,6 +15,8 @@ const TRITON_ENVIRONMENTS = {
     SFTP_PORT: "",
     ERROR_MONITORING_EMAIL: "",
     TECH_SUPPORT_EMAIL: "",
+    TO_VALIDATION_EMAIL: "sequencing-runs@computationalgenomics.ca",
+    TO_TEST_EMAIL: "andres.tocasuche@c3g.ca",
```

Comment:
I'd argue that `TO_TEST_EMAIL` is unnecessary. We can stick to just one `TO_VALIDATION_EMAIL`.

Reply:
Sebastian Amouzegar
  1:56 PM
ah you mean to use a different email than sequencing-runs@computationalgenomics.ca
1:56
and you would like to send the test emails where?


Nafiz Islam
  [1:56 PM]
andres.tocasuche@c3g.ca for validation (edited)

where do you want to add the email to send the tests?